### DISCUSSION TITLE
Several similar modules in a server/device

### DISCUSSION THREAD

**Discussion by dasfliege** [netbox-community/netbox#15051] [Help Wanted!] [answered]
We try to integrate our new servers into netbox. They all have three PCIe slots, each populated with a 2-Port SFP28 adapter. I created a module type for the adapter. If i try to assign the second module to the server module bay 2, i receive the following error: "Interface - SFP28 Port 1 already exists".
I wonder how this is meant to be done in netbox. I guess it's a quite common scenario that a server has more then one module of the same type. Do i really have to create three different module types with different port descriptions, even though all of them are the exact same?

**Comment by candlerb** [comment, marked as answer]
You need to use the placeholder `{module}` as part of the interface name in the interface template.

Admittedly a bit tricky to find in the documentation, but it's in two places:
* https://docs.netbox.dev/en/stable/models/dcim/moduletype/#automatic-component-renaming
* https://docs.netbox.dev/en/stable/features/devices-cabling/#module-types-modules

**Reply by dasfliege** [reply]
That worked perfectly fine. Thanks a lot!
